tests/integration/framework/process/scheduler/etcd/etcd.go

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Stopping review now as there seems to be some broken history or cherry pick causing the diff to become wrong.